feature-libs/product-configurator/rulebased/components/price/configurator-price.component.ts

```diff
@@ -41,7 +48,11 @@ export class ConfiguratorPriceComponent {
    * @return {string} - total price formula
    */
   get priceTotal(): string {
-    return '+ ' + this.formula.priceTotal?.formattedValue;
+    if (this.formula.priceTotal && this.formula.priceTotal.value >= 0) {
+      return '+' + this.formula.priceTotal?.formattedValue;
+    } else {
+      return this.formula.priceTotal?.formattedValue || '';
+    }
   }
 
```

Comment:
I'm a bit confused - `get price` and `get priceTotal` behave different for prices with value '0'.
 `get price` returns  '0'
 `get priceTotal` returns  '+0'
is this intended?

Reply:
Agree!